README.md

```diff
@@ -3,46 +3,138 @@
 Concourse is an open-source continuous thing-doer.
 Built on the simple mechanics of resources, tasks, and jobs, Concourse presents a general approach to automation that makes it great for CI/CD.
 
-## Architecture
+# Architecture
 
 This stack will deploy a Concourse server (web+atc) on an Amazon EC2 instance behind ALB/NLB loadbalancers, using PostgreSQL for Cycloid.io.
 
-![](https://cdn.jsdelivr.net/gh/cycloid-community-catalog/stack-concourse-server@28b0425/diagram.svg)
+<p align="center">
+<img src="docs/diagram.png" width="400">
+</p>
 
   * **ALB**: Amazon application loadbalancer
   * **NLB**: Amazon network loadbalancer
   * **ASG**: Autoscaling group for fronts
   * **concourse-server**: EC2 instances from builded AMI
   * **RDS**: Amazon RDS database (PostgreSQL)
 
-> **Pipeline** The pipeline contains a manual approval between terraform plan and terraform apply.
-> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
-> and click on the `+` button to trigger it.
-
-## Requirements
+# Requirements
 
 In order to run this task, couple elements are required within the infrastructure:
 
-  * Having a VPC with private & public subnets containing a bastion server that can access instances by SSH
-  * Having an S3 bucket for terraform remote states
-
-## Details
+  * Having a VPC with private & public subnets [Here](https://docs.aws.amazon.com/vpc/latest/userguide/getting-started-ipv4.html#getting-started-create-vpc)
+  * Having an S3 bucket to store Terraform remote states [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
 
-**Pipeline**
+# Details
 
-  * Run packer to build debian Amazon AMI with Concourse server exposing telegraph metrics (port 9100 prometheus format).
-  * Clean old builded AMI
+## Pipeline
 
-**Terraform**
-
-Create:
-
-  * ALB (application loadbalancer), but you can use your own ALB
-  * NLB (network loadbalancer) for the worker SSH access
-  * ASG with launch_template of a concourse server
-  * RDS
+> **Note** The pipeline contains a manual approval between terraform plan and terraform apply.
+> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
+> and click on the `+` button to trigger it.
 
-**Ansible**
+<img src="docs/pipeline.png" width="800">
+
+**Jobs description**
+
+  * `terraform-plan`: Terraform job that will simply make a plan of the stack.
+  * `terraform-apply`: Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
+  * `terraform-destroy`: :warning: Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed. Use with caution.
+  * `build-ami` : Build Concourse server Amazon image AMI using Packer and Ansible.
+  * `clean-ami` : Remove old AMI of Cycloid workers.
+
```

Comment:
I'd reorder by jobs order.
The deploy AMI isn't present here, is it still required after an apply, or forgotten?